File: FundamentosPython/Aula 04 - Condicoes Aninhadas/triangulo.py
# Desenvolva um programa em Python que solicite três valores inteiros representando os lados de um triângulo, e informe se o triângulo é 

# Equilátero (todos os lados iguais)
# Isósceles (dois lados iguais) 
# Escaleno (todos diferentes).


ladoA = int(input("Digite um lado: ")) 
laboB = int(input("Digite um lado: "))
ladoC = int(input("Digite um lado: "))

if ladoA == laboB and ladoA == ladoC  and laboB == ladoC:
    print("TRIÂNGULO EQUILÁTERO")
elif ladoA == laboB or ladoA == ladoC or ladoC == laboB:  
    print("TRIÂNGULO ISÓCELES")
elif ladoA != laboB and ladoA != ladoC and laboB != ladoC:
    print("TRIÂNGULO ESCALENO")
    

File: DesenvolvimentoJavaScript/Aula11-CondicoesEncadeadas/operadoresLogicos/triangulo.js
let ladoA = 10
let ladoB = 20
let ladoC = 100

let ladoAB = ladoA + ladoB 
let ladoAC = ladoA + ladoC
let ladoBC = ladoB + ladoC

if (ladoAB > ladoC && ladoAC > ladoB && ladoBC > ladoA){
    console.log("LADOS VÁLIDOS PARA FORMAR UM TRIÂNGULO");
    if (ladoA == ladoB == ladoC) {
        console.log("EQUILÁTERO");
    } else if (ladoA == ladoB || ladoA == ladoC || ladoB == ladoC){
        console.log("ISÓSCELES");
    } else if (ladoA != ladoB != ladoC){
        console.log("ESCALENO");
    }
} 
else {
    console.log("NÃO FOI POSSÍVEL FORMAR UM TRIÂNGULO");
}